src/docs/co2js/tutorials/getting-started-browser.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Getting started: In the browser"
+description: "In this tutorial, you will use CO2.js in the browser to calculate the CO2 emissions of transferring 1 gigabyte (GB) of data."
+relatedPR: https://github.com/thegreenwebfoundation/co2.js/pull/92
+eleventyNavigation:
+  key: getting-started-browser
+  title: "Getting started: In the browser"
+  # parent: overview
+  sectionTitle: Tutorials
+  order: 10
+---
+# {{ title }}
+
+## Overview
+
+In this tutorial, you will use CO2.js in the browser to calculate the CO2 emissions of transferring 1 gigabyte (GB) or data.
```

Comment:
The only thing I might add is to be explicit about the model we're using to work out the figures.

Can you be a tad more explicit about us using the 1byte model based on the work by the Shift Project in France in this part?

It may also make sense to refer to the ability to swap in different models, but I'm happy to leave this decision to you.

Reply:
Thanks Chris. I've added a note about the default model & the planned work to update it to SWD in https://github.com/thegreenwebfoundation/developer-docs/pull/5/commits/3f224eca84f9255e04300c9d176241f0ea4f1a83

Regarding the ability to swap, I'll update the docs for that once we've got an API for that in place (planned for v0.11.0)